data/data/aws/main.tf

```diff
@@ -21,6 +21,7 @@ module "bootstrap" {
   target_group_arns        = "${module.vpc.aws_lb_target_group_arns}"
   target_group_arns_length = "${module.vpc.aws_lb_target_group_arns_length}"
   vpc_security_group_ids   = "${list(module.vpc.master_sg_id)}"
+  vpc_id                   = "${module.vpc.vpc_id}"
```

Comment:
This should sort before `vpc_security_group_ids` (also in `variables.tf`).  You probably also need to make this variable optional (and not create the special security groups if it is unset) or update the stand-alone example in `data/data/aws/bootstrap/README.md`.

Reply:
> This should sort before vpc_security_group_ids (also in variables.tf). 

done with https://github.com/openshift/installer/pull/1105/commits/a2de5d75cb2af5dd2c7803172e9b5eaf7604f7ec -> https://github.com/openshift/installer/pull/1105/commits/93d9a412ff674079f90b2732d6ca4ac4d5d2389b

>  You probably also need to make this variable optional (and not create the special security groups if it is unset) or update the stand-alone example in data/data/aws/bootstrap/README.md.

keeping it required and updated the readme with https://github.com/openshift/installer/pull/1105/commits/a2de5d75cb2af5dd2c7803172e9b5eaf7604f7ec -> https://github.com/openshift/installer/pull/1105/commits/93d9a412ff674079f90b2732d6ca4ac4d5d2389b